src/docs/get-started/install/_path-mac-linux.md

```diff
@@ -5,14 +5,20 @@ the command line, as shown in [Get the Flutter SDK](#get-sdk).
 You'll probably want to update this variable permanently,
 so you can run `flutter` commands in any terminal session.
 
-The steps for modifying this variable permanently for all terminal
-sessions are machine-specific.
-Typically you add a line to a file that is executed whenever you open
-a new window. For example:
+The steps for modifying this variable permanently for
+all terminal sessions are machine-specific.
+Typically you add a line to a file that is executed
+whenever you open a new window. For example:
 
  1. Determine the directory where you placed the Flutter SDK.
     You will need this in Step 3.
- 2. Open (or create) `$HOME/.bash_profile`. The file path and filename might be
+ 2. Open (or create) the `rc` file for your shell.
+    For example, Linux and Mac OS Mojave (and earlier)
+    use the Bash shell by default,
+    so edit `$HOME/.bashrc`.
+    Mac OS Catalina uses the Z shell by default,
+    so edit `$HOME/.zshrc`.
+    The file path and filename might be even
```

Comment:
Instead of "The file path and filename might be even different on your machine", which sounds kind of vague, can we say "If you're using a different shell, the file path and filename will be different on your machine"?